assets/src/edit-story/elements/video/output.js

```diff
@@ -34,6 +34,7 @@ function VideoOutput({ element, box }) {
 
   const poster = defaultForUndefined(element.poster, resource.poster);
   const alt = defaultForUndefined(element.alt, resource.alt);
+  const isMuted = defaultForUndefined(element.isMuted, resource.isMuted);
```

Comment:
`isMuted` is always set on the `resource`, never on the `element`, so this not accurate nor needed. Can just use `resource.isMuted`

Reply:
Fixed.